src/Fable.Transforms/Rust/Fable2Rust.fs

```diff
@@ -1566,7 +1577,7 @@ module Util =
             else
                 fullName
 
-    let makeUnion (com: IRustCompiler) ctx r values tag entRef genArgs =
+    let makeUnion (com: IRustCompiler) ctx r isStruct values tag entRef genArgs =
```

Comment:
@alexswan10k Please remove change as `isCopyableEntity` already handles value types correctly.

Reply:
Agree for unions. Not sure how I ended up here. Unions appear to work! Fixed.